### PR TITLE
Restructure the project.

### PR DESCRIPTION
**Pull request by shottah**
This fixes the gitignore file, and strucutres the project in the simplest manner.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# "mu-plugins", "plugins", "themes" directory
+!/wp-content/mu-plugins/
+!/wp-content/plugins/
+!/wp-content/themes/
+
+# ignore these plugins
+/wp-content/plugins/hello.php
+
+# ignore log files and databases
+*.log
```
Comment (shottah) [inline .gitignore]: This will upload *.SQL files right?

**Review by shottah — COMMENTED** [review]
@asayee0 have you been able to create a fresh site, copy and replace the contents of this branch into the project and it work?

were you able to push and make changes after that and it still worked?

**Comment by asayee0** [comment]
Ignore my last commit comment that says that the twenty* themes were ignored. I'll work on that later.

**Comment by shottah** [comment]
Seen.

**Comment by shottah** [comment]
Remember that gitignore will not change the files that are already in the
repo it specifically works to ignore what is picked up when you run
commands like git rm git add and git status.

**Pull request merged**